Dockerfile

```diff
@@ -22,14 +25,18 @@ RUN yum groupinstall -y -q "Development Tools"
 
 # Install Compass
 RUN gem install json_pure
-RUN gem update --system
+RUN ruby --version
+RUN gem install "rubygems-update:<3.0.0" --no-document
+RUN update_rubygems
+RUN ruby --version
+RUN gem install rb-inotify -v 0.9.10
+RUN gem install compass
 
 # Install the latest version of git
 RUN cd /tmp && \
-    wget https://github.com/git/git/archive/v2.7.0.tar.gz && \
-    tar xvfz ./v2.7.0.tar.gz && \
-    cd git-2.7.0 && \
-    make configure && \
+    wget https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.9.5.tar.gz && \
+    tar xvfz ./git-2.9.5.tar.gz && \
+    cd git-2.9.5 && \
```

Comment:
You don't need the make configure anymore?

Reply:
not anymore

Reply:
I remember that cause the error on centos6, I'll check if that works with `make config`